frontends/precinct-scanner/src/components/export_backup_modal.tsx

```diff
@@ -166,7 +166,7 @@ export function ExportBackupModal({ onClose, usbDrive }: Props): JSX.Element {
   }
 
   if (currentState === ModalState.SAVING) {
-    return <Modal content={<Loading />} onOverlayClick={onClose} />;
+    return <Modal content={<Loading>Saving Backup</Loading>} />;
```

Comment:
Here is the only non-copy change of the PR. Actually adding copy ("Saving Backup") rather than "Loading", and preventing close by overlay click.

Reply:
Thanks for pointing this out!